### PR TITLE
Add templates for v11.1

### PR DESCRIPTION
**Pull request by Novkirishki**

### CODE REVIEW

File: Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+...
+
+main.scss
+//Import Bootstrap from npm
+@import "../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap.scss";
+@import "../../../../node_modules/magnific-popup/src/css/main.scss";
+
+// Sitefinity
+@import "../../sitefinity/sass/components/icons/sf-icon-font";
+@import "../../sitefinity/sass/widgets/socialShare/sf-sprite";
```
Comment (jordanilchev) [inline Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/README.md]: do we still need this? @JenPeleva

File: Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-social-share.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.sf-social-share {
```
Comment (jordanilchev) [inline Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-social-share.scss]: do we still need this?

File: Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-sprite.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
+.sf-icon-blogger {
```
Comment (jordanilchev) [inline Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-sprite.scss]: do we still need this

File: Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/README.md (hunk 1 above)
Comment (nikistefanov) [reply Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/README.md]: @Novkirishki @jordanilchev We don't need this. Could you please get the latest templates.

File: Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-social-share.scss (hunk 2 above)
Comment (nikistefanov) [reply Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-social-share.scss]: Same as above.

File: Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-sprite.scss (hunk 3 above)
Comment (nikistefanov) [reply Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/assets/src/sitefinity/sass/widgets/socialShare/_sf-sprite.scss]: Same as above.

**Pull request merged**